seismic/inventory/engd2stxml.py

```diff
@@ -26,6 +26,7 @@
 import argparse
 import datetime
 
+from ordered_set import OrderedSet as set
```

Comment:
I would prefer this be reverted, as this module does not need an ordered set.

Reply:
Are you certain that results across multiple runs would be identical with Py3? Having that consistency with results helps with debugging/diffing etc.

Reply:
In this case, yes, because the ordering of the set contents is irrelevant. Using an ordered set implies that it might be relevant. In cases where I want to order the contents (e.g. for printing to stdout), then I will convert to `list` and call `sort`. But the container itself does not need to be implicitly ordered, and any variation in ordering will not effect test results.